src/pages/user/index.js

```diff
@@ -84,6 +84,7 @@ const UserRules = ({ data, location }) => {
     return name
       .normalize('NFD')
       .replace(' [SSW]', '')
+      .replace('ø', 'oe')
```

Comment:
While I think this fixes the issue, it won't stop this happening again with someone else with a unicode character in their name. maybe we could use https://www.npmjs.com/package/unidecode ?

Reply:
Hi Jack,
Good point! In the code we have `name.normalize('NFD').replace(/\p{Diacritic}/gu, '')` which can remove accents/diacritics in some names (like áèéç, etc) . But the character "ø" is so special that I think we still have to handle it manually in this case.
I have tried out the unidecode and other packages, the `ø` can only be transliterated into `o`. 🤦‍♀️

Reply:
ah ok interesting... 🙃🙃🙃